packages/hardhat-plugin/tests/hardhat-mock-projects/eth_getTransactionCount-random-value-project/hardhat.config.ts

```
@@ -29,8 +29,8 @@
     },
 
     // Custom parameters
-    delegator: {
-        delegatorPrivateKey:
+    delegator: { // TO BE RENAMED WHEN BNC REFACTORING DONE
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)